requirements-pi.txt

```diff
@@ -1,52 +1,55 @@
-protobuf==3.20.1; python_full_version >= "3.7.1" and python_version >= "3.7" and python_full_version < "4.0.0" and (python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.7")
-pyasn1-modules==0.2.8; python_full_version >= "3.7.1" and python_version >= "3.6" and python_full_version < "4.0.0" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6")
-pyasn1==0.4.8; python_full_version >= "3.7.1" and python_version >= "3.6" and python_full_version < "4.0.0" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6") and python_version < "4"
-pycparser==2.21; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.6" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6")
-pyparsing==3.0.9; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.6"
-pyrsistent==0.18.1; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.7"
-pyserial==3.5
-python-dateutil==2.8.2; python_full_version >= "3.7.1" and python_full_version < "4.0.0"
-python-dotenv==0.20.0; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.6"
-python-slugify==5.0.2; python_version >= "3.6"
-pyyaml==6.0; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.6"
-requests==2.27.1; python_full_version >= "3.7.1" and python_version >= "3.6" and python_full_version < "4.0.0"
-rsa==4.8; python_version >= "3.6" and python_version < "4" and python_full_version >= "3.7.1" and python_full_version < "4.0.0" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6")
-six==1.16.0; python_full_version >= "3.7.1" and python_version >= "3.6" and python_full_version < "4.0.0" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6")
-text-unidecode==1.3; python_version >= "3.6"
-twined==0.5.0; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.6"
-typing-extensions==4.2.0; python_version < "3.8" and python_version >= "3.7" and python_full_version >= "3.7.1" and python_full_version < "4.0.0"
-urllib3==1.26.9; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version < "4" and python_version >= "3.6"
-werkzeug==2.1.2; python_full_version >= "3.7.1" and python_full_version < "4.0.0" and python_version >= "3.7"
-zipp==3.8.0; python_version < "3.8" and python_version >= "3.7" and python_full_version >= "3.7.1" and python_full_version < "4.0.0"
+attrs==22.1.0 ; python_full_version >= "3.7.1" and python_version < "3.10"
```

Comment:
Update this and the dev equivalent if @thclark confirms it's ok to drop `python3.7` support

Reply:
@thclark please could you comment on this?